lib/src/mock_query.dart

```diff
@@ -163,6 +168,21 @@ class MockQuery extends Mock implements Query {
       } else {
         return false;
       }
+    } else if (whereIn != null) {
+      if (whereIn.length > 10) {
+        throw ArgumentError(
+          'whereIn cannot contain more than 10 comparison values',
+        );
+      }
+      if (arrayContainsAny != null) {
+        throw FormatException(
+          'whereIn cannot be combined with arrayContainsAny',
+        );
+      }
+      if (whereIn.contains(value)) {
```

Comment:
A more concise way would be to write `return whereIn.contains(value);` directly.

Reply:
Totally true. Looks like I wrote it half awake 😴